x-pack/plugins/ml/server/models/data_frame_analytics/analytics_manager.ts

```diff
@@ -121,10 +157,14 @@ export class AnalyticsManager {
 
   private findJobModel(analyticsId: string): any {
     return this.inferenceModels.find(
-      (model: any) => model.metadata?.analytics_config?.id === analyticsId
+      (model) => model.metadata?.analytics_config?.id === analyticsId
     );
   }
 
+  private findJobStats(analyticsId: string): any {
```

Comment:
please define a return type for the job stats

Reply:
Updated in 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9